source/adapters/level_zero/CMakeLists.txt

```diff
@@ -22,8 +22,25 @@ endif()
 if (NOT DEFINED LEVEL_ZERO_LIBRARY OR NOT DEFINED LEVEL_ZERO_INCLUDE_DIR)
     message(STATUS "Download Level Zero loader and headers from github.com")
 
+    # Workaround warnings/errors for Level Zero build
+    set(CMAKE_CXX_FLAGS_BAK "${CMAKE_CXX_FLAGS}")
+    if (WIN32)
+        # FIXME: Level Zero build fails with /DUNICODE
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /UUNICODE")
```

Comment:
this is set below with `target_compile_options`.

Reply:
thanks @pbalcer . Done.